README.md

```diff
@@ -31,10 +31,15 @@ curl -s https://static.rust-lang.org/rustup.sh | sh -s -- --channel=nightly
 
 # 2. Install RocksDB Library
 
+# 2.1 [optional step] Upgrade Your GCC To Version At Least 4.7 To Get C++11 Support
```

Comment:
I guess you may just add a reference to rocksdb's install guaid. https://github.com/facebook/rocksdb/blob/master/INSTALL.md